acceptance/openstack/sharedfilesystems/v2/sharenetworks_test.go

```diff
@@ -21,3 +22,66 @@ func TestShareNetworkCreateDestroy(t *testing.T) {
 
 	defer DeleteShareNetwork(t, client, shareNetwork)
 }
+
+func TestShareNetworkList(t *testing.T) {
```

Comment:
I suspect if you had a pagination acceptance test (instead of calling `AllPages`) with more than 1 page, you'd see if the `LinkedPageBase` is correct (I suspect it isn't).